src/style/App.css

```diff
@@ -50,13 +110,171 @@ body {
   padding: 2rem;
 }
 
+.book-info {
+  flex: 2;
+  display: flex;
+  flex-direction: column;
+  gap: 1rem;
+}
+
+.book-category {
+  font-weight: 700;
+  font-family: 'Montserrat', sans-serif;
+  font-size: 0.875rem;
+  color: #777;
+}
+
+.book-title {
+  text-transform: capitalize;
+  font-weight: 700;
+  font-size: 1.375rem;
+}
+
+.book-author {
+  font-weight: 300;
+  text-transform: capitalize;
+  font-size: 0.875rem;
+  color: var(--blue);
+}
```

Comment:
Kindly ensure that the style of the details is matching the design guide. Kindly note that "Comments" is vertically aligned with the name and author of the book. Also, there's a lesser margin on the design between the text above and below the title.

Your design:
![image](https://user-images.githubusercontent.com/20554583/216529150-519aebd2-c24b-4782-8da3-54d433111f11.png)

Design guide:
![image](https://user-images.githubusercontent.com/20554583/216529179-ca84f2cb-83cc-4997-9e7d-2f6fee0d3dfd.png)